ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -832,19 +828,15 @@ periodics:
         chmod ug+x /tmp/bin/oc
 
         # annotate the information about this job so that we can query for it
-        oc --kubeconfig /etc/appci/sa.release-bot.app.ci.config annotate pj/${PROW_JOB_ID} "release.openshift.io/from-tag=${from}" "release.openshift.io/tag=${to}" "release.openshift.io/source=ocp/$(BRANCH)"
+        oc --kubeconfig /etc/appci/sa.release-bot.app.ci.config annotate pj/${PROW_JOB_ID} "release.openshift.io/from-tag=${from}" "release.openshift.io/tag=${to}" "release.openshift.io/source=ocp/4.1"
 
         ci-operator $@
       - ""
       env:
-      - name: BRANCH
-        value: "4.1"
-      - name: CLUSTER_TYPE
-        value: aws
```

Comment:
My thought about this is whether we are sure there isn't any tool that expect these to be present so that it can overwrite them. I remember this was the case with `RELEASE_IMAGE_*` variables in the past, where release-controller expected it to be present in the job configuration so it can set its value. When we removed it from the config, it broke some things. Are we sure this is safe?

Reply:
Good question. I would wager yes, but I'll check. I don't think it's valid to change the branch, for instance.

Reply:
Checked, we are good